src/eko/anomalous_dimensions/aem1.py

```diff
@@ -0,0 +1,88 @@
+# -*- coding: utf-8 -*-
+import numba as nb
+
+from .. import constants
+from . import as1
+
+
+@nb.njit("c16(c16)", cache=True)
+def gamma_phq_0(N):
```

Comment:
Here (and related) the agreement was to drop `_0`, since there is no consistent way to assign it, and rely on the module scope.

Reply:
Related: all the functions in this subpackage, not only the same module (i.e. the same file).